docs/reference/ccr/getting-started.asciidoc

```diff
@@ -218,20 +236,20 @@ To enable soft deletes on indices created on versions of
 
 [[ccr-getting-started-follower-index]]
 ==== Create a follower index
```

Comment:
I wonder if we want to cast this as "Create a follower index to replicate a content index" vs "Create an autofollow pattern to replicate time series indices"?